birdhouse/env.local.example

```diff
@@ -409,6 +409,28 @@ export GEOSERVER_ADMIN_PASSWORD=geoserverpass
 # Name of organization hosting the Thredds server
 #export THREDDS_ORGANIZATION="Birdhouse"
 
+# Additional catalogs for THREDDS. Add as many datasetScan XML blocks as needed to THREDDS_ADDITIONAL_CATALOG.
+# Each block defines a new top-level catalog. See birdhouse/config/thredds/catalog.xml.template for more information.
+export THREDDS_ADDITIONAL_CATALOG=""
+#export THREDDS_ADDITIONAL_CATALOG="
+#    <datasetScan name="dataset_location_name" ID="dataset_url_path" path="dataset_url_path" location="dataset_location_on_container">
+#
+#      <metadata inherited="true">
+#        <serviceName>all</serviceName>
+#      </metadata>
+#
+#      <filter>
+#        <include wildcard="*.nc" />
```

Comment:
If you actually uncomment his example code, does it work?  No need to escape all the double quotes?

Reply:
Oh right... I haven't tested. I think just replacing it with single quote is fine right?

Reply:
Yes I think single quote would do the trick.  Easy, just uncomment that sample override to test.

Reply:
Fix all problems. Tested to check the feature works.